src/rprblender/export/mesh.py

```diff
@@ -405,6 +438,12 @@ def sync(rpr_context: RPRContext, obj: bpy.types.Object, **kwargs):
 
         if data.vertex_colors is not None:
             rpr_shape.set_vertex_colors(data.vertex_colors)
+            
+        if data.vertex_attributes is not None:
+            for att in data.vertex_attributes:
+                if att[0] not in rpr_context.mesh_attribute_names:
+                    rpr_context.mesh_attribute_names.append(att[0])
```

Comment:
Personally I would do it such that the mesh_attribute_names is a dictionary where the key:value is 

attribute_name : index

Reply:
I'm not entirely sure of the proper way to go about this.

Reply:
Probably it's a good idea to use `key: value`, where `key` is the combination of a mesh key and attribute name, and the `value` is an index.
To make it easy to access attributes of a specific mesh during sync_update is the purpose.